data/zilliqa/assets/zil1kh0gj83h2teh7ycatkgq27d4agtug00t56d9ml/meta.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "The ZilOwls",
+  "symbol": "ZILO",
+  "links": {
+    "twitter": "https://twitter.com/eblkronos"
+  },
+    "config": {
```

Comment:
the indentation here isn't correct (2 less in front of config, and the lines below that too except the end brackets)

Reply:
cc @SpardaNFTs just so you see